base_changeset/models/record_changeset.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2015-2017 Camptocamp SA
+# Copyright 2020 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class RecordChangeset(models.Model):
+    _name = "record.changeset"
+    _description = "Record Changeset"
+    _order = "date desc"
+    _rec_name = "date"
+
+    model = fields.Char(index=True, required=True, readonly=True)
+    res_id = fields.Integer(
+        string="Record ID",
+        index=True,
+        required=True,
+        readonly=True,
+    )
+    change_ids = fields.One2many(
+        comodel_name="record.changeset.change",
+        inverse_name="changeset_id",
+        string="Changes",
+        readonly=True,
+    )
+    date = fields.Datetime(
+        string="Modified on", default=fields.Datetime.now(), index=True, readonly=True
```

Comment:
Could it be that the callable that was intended?
```suggestion
        string="Modified on", default=fields.Datetime.now, index=True, readonly=True
```

Reply:
oh yes, this seems wrong, thanks! @astirpe are we missing something fancy?